polyfill/lib/absolute.mjs

```diff
@@ -107,11 +107,11 @@ export class Absolute {
     ns = twons - onens;
     ms = twoms - onems;
 
-    const duration = new ES.GetIntrinsic('%Temporal.Duration%')(delta);
+    const duration = ES.GetIntrinsic('%Temporal.Duration%')(delta);
```

Comment:
perhaps just this would be clearer?
```suggestion
    const Duration = ES.GetIntrinsic('%Temporal.Duration%');
    const duration = new Duration(delta);
```